src/app/component/user/components/habit/all-habits/all-habits.component.ts

```diff
@@ -1,10 +1,14 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
-import { Subscription } from 'rxjs';
+import { BehaviorSubject, Subscription} from 'rxjs';
 
 import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
-import { ServerHabitItemPageModel } from '@global-user/models/habit-item.model';
-import { AllHabitsService } from './services/all-habits.service';
+import { HabitService } from '@global-service/habit/habit.service';
+import { map } from 'rxjs/operators';
+import { HabitInterface, HabitListInterface } from '../../../../../interface/habit/habit.interface';
+import { singleNewsImages } from '../../../../../image-pathes/single-news-images';
+import {HabitAssignService} from '@global-service/habit-assign/habit-assign.service';
```

Comment:
Remove this unused import of 'HabitAssignService'

Reply:
fixed